### PR TITLE
fix: typo in InterceptResolutionStrategy

### PR DESCRIPTION
**Pull request by harshagrawal523**
**What kind of change does this PR introduce?**
 There was a typo in line 210.

**Did you add tests for your changes?**
No

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**
None

### CODE REVIEW

**Comment by FdezRomero** [comment]
Hey @harshagrawal523! I noticed the same typo and was going to open a PR when I saw this one 😄

Can you sign the CLA and fix the commit message to follow the Conventional Commits standard so the tests can run? (e.g. `fix: typo in InterceptResolutionStrategy`). Thanks!

**Comment by harshagrawal523** [comment]
> 

Hi , actually this was my first PR , I just signed my CLA , what should I do next?

**Comment by FdezRomero** [comment]
@harshagrawal523 I think a maintainer has to approve the workflow, since it's your first PR. Let's wait and see 👀 

Update: This typo has been also fixed in #7796.

**Comment by benallfree** [comment]
I recommend closing this for #7796 or please update the Typescript typing as well 👍

File: src/common/HTTPRequest.ts [hunk 1]
```diff
@@ -216,7 +216,7 @@ export class HTTPRequest {
    */
   private interceptResolution(): [InterceptResolutionStrategy, number?] {
     if (!this._allowInterception) return ['disabled'];
-    if (this._interceptionHandled) return ['alreay-handled'];
+    if (this._interceptionHandled) return ['already-handled'];
```
Comment (benallfree) [inline src/common/HTTPRequest.ts:219]: The `InterceptResolutionStrategy` must be updated too.

**Review by benallfree — CHANGES_REQUESTED** [review]

File: src/common/HTTPRequest.ts (hunk 1 above)
Comment (harshagrawal523) [reply src/common/HTTPRequest.ts:219]: ohh , I will check it once

**Comment by benallfree** [comment]
Fixed in #7813

**Pull request closed without merging**